### ISSUE TITLE
Idea: turn neoformat into an operator

### ISSUE THREAD

**Issue by Alok** [sbdchd/neoformat#67]
basically make neoformat into an operator mapping like `gq` improved.

**Comment by handijk** [comment]
Yes, being able to use it as a drop in replacement for something like = would be very cool. (i never used gq)

**Comment by lpil** [comment]
Most formatters will not be able to format a portion of a file correctly as they require the context to understand how to format, or to even make the syntax valid.

See this issue an example of this causing problems: https://github.com/elixir-editors/vim-elixir/issues/401
